introduction/report-structure.md

```diff
@@ -0,0 +1,18 @@
+# Report structure
+To anwser the four main subquestions, the report will have four sections.  
+The first subquestion zooms in on what can be learned from (non-software) communities.
+Before theories can be defined about what can be learned from all sorts of communities, it must be defined what a community is.
+Because communities exist in different sizes, with different structures and on different platforms, a few of these communities will be analysed in section 1.
+
+Software development in its current form has some methodologies on how requirements are established for a (big) software development project.
+*** referentie naar subvraag ***  
+These requirements are the basis of how the software program will be build and are therefore an essential part of the software development process. In the context of this project, the requirements are build from the problem-description and thus must be correctly translated.
```

Comment:
`In the context of this project, the requirements are build from the problem-description and thus must be correctly translated.`

Again, built instead of build, but I think this sentence can be left out entirely. There isn't really a need to make claims here about requirement translation in my opinion